framework/src/reporters/ElementStatistics.C

```diff
@@ -0,0 +1,81 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ElementStatistics.h"
+
+InputParameters
+ElementStatistics::validParams()
+{
+  InputParameters params = ElementReporter::validParams();
+  params.addParam<std::string>("base_name", "Name to append to reporters.");
+  return params;
+}
+
+ElementStatistics::ElementStatistics(const InputParameters & parameters)
+  : ElementReporter(parameters),
+    _base_name(isParamValid("base_name") ? getParam<std::string>("base_name") + "_" : ""),
+    _max(declareValueByName<Real>(_base_name + "max")),
+    _min(declareValueByName<Real>(_base_name + "min")),
+    _average(declareValueByName<Real>(_base_name + "average")),
+    _integral(declareValueByName<Real>(_base_name + "integral")),
+    _number_elements(declareValueByName<int>(_base_name + "number_elements"))
+{
+}
+void
+ElementStatistics::initialize()
```

Comment:
I'm surprised formatting didn't pick this up
```suggestion
{
}

void
ElementStatistics::initialize()
```